From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: fukiame <fukiame@proton.me>
Date: Fri, 14 Aug 2026 13:30:06 +0700
Subject: [PATCH] Revert "Add notifyPaused method to AutoLibrarySyncService"

This reverts commit 9297fe16d5ff00f076903142dfc73928958a8760.

diff --git a/lib/services/auto_library_sync_service.dart b/lib/services/auto_library_sync_service.dart
index 08cf2493..7ec830e3 100644
--- a/lib/services/auto_library_sync_service.dart
+++ b/lib/services/auto_library_sync_service.dart
@@ -35,11 +35,6 @@ class AutoLibrarySyncService {
     devLog('Auto library sync stopped');
   }
 
-  void notifyPaused() {
-    if (!_isRunning) return;
-    _observerService?.notifyPaused();
-  }
-
   void notifyResumed() {
     if (!_isRunning) return;
     _observerService?.notifyResumed();
-- 
2.55.0

