From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: fukiame <fukiame@proton.me>
Date: Tue, 28 Jul 2026 14:46:24 +0700
Subject: [PATCH] yeah

Signed-off-by: fukiame <fukiame@proton.me>

diff --git a/lib/widgets/navigation/flick_nav_bar.dart b/lib/widgets/navigation/flick_nav_bar.dart
index edf27e9f..12bc8f40 100644
--- a/lib/widgets/navigation/flick_nav_bar.dart
+++ b/lib/widgets/navigation/flick_nav_bar.dart
@@ -104,8 +104,7 @@ class _FlickNavBarState extends State<FlickNavBar> {
         children: [
           if (sw) ...[ widget.miniPlayerWidget!,
                         Text(
-                          widget.miniPlayerWidget!.toStringShort(),
-			  maxLines: 1,
+                          widget.miniPlayerWidget!.toString(),
                           overflow: TextOverflow.ellipsis,
                           style: TextStyle(
                             fontFamily: 'ProductSans',
@@ -116,8 +115,18 @@ class _FlickNavBarState extends State<FlickNavBar> {
                           ),
                         ),
                         Text(
-                          ( widget.miniPlayerWidget! == const Offstage(offstage: true) ).toString(),
-			  maxLines: 1,
+                          widget.miniPlayerWidget!.toStringShallow(),
+                          overflow: TextOverflow.ellipsis,
+                          style: TextStyle(
+                            fontFamily: 'ProductSans',
+                            fontSize: 14,
+                            fontWeight: FontWeight.w400,
+                            color: Colors.white,
+                            letterSpacing: 0.4,
+                          ),
+                        ),
+                        Text(
+                          widget.miniPlayerWidget!.toStringDeep(),
                           overflow: TextOverflow.ellipsis,
                           style: TextStyle(
                             fontFamily: 'ProductSans',
-- 
2.55.0

