From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: fukiame <fukiame@proton.me>
Date: Fri, 14 Aug 2026 13:30:06 +0700
Subject: [PATCH] Revert "Add setExtendedVolumeEnabled method"

This reverts commit 56a01ba653bd8cedf9a7e55254fae84fdc49158b.

diff --git a/lib/providers/app_preferences_provider.dart b/lib/providers/app_preferences_provider.dart
index c76b88657..c7e094025 100644
--- a/lib/providers/app_preferences_provider.dart
+++ b/lib/providers/app_preferences_provider.dart
@@ -119,14 +119,6 @@ class AppPreferencesNotifier extends Notifier<AppPreferences> {
     await ref.read(appPreferencesServiceProvider).setCrossfadeCurveIndex(value);
   }
 
-  Future<void> setExtendedVolumeEnabled(bool value) async {
-    if (state.extendedVolumeEnabled == value) return;
-    state = state.copyWith(extendedVolumeEnabled: value);
-    await ref
-        .read(appPreferencesServiceProvider)
-        .setExtendedVolumeEnabled(value);
-  }
-
   Future<void> setSwipeActionsEnabled(bool value) async {
     if (state.swipeActionsEnabled == value) return;
     state = state.copyWith(swipeActionsEnabled: value);
-- 
2.55.0

