From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: fukiame <fukiame@proton.me>
Date: Fri, 14 Aug 2026 13:30:06 +0700
Subject: [PATCH] Revert "Use player max volume in volume slider"

This reverts commit b161cfd8544bbf401b73fda1399c7136983f918f.

diff --git a/lib/features/player/widgets/volume_bottom_sheet.dart b/lib/features/player/widgets/volume_bottom_sheet.dart
index c96d84ace..78c40465f 100644
--- a/lib/features/player/widgets/volume_bottom_sheet.dart
+++ b/lib/features/player/widgets/volume_bottom_sheet.dart
@@ -18,8 +18,7 @@ class VolumeBottomSheet extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    final maxVolume = playerService.maxVolume;
-    final initialVolume = playerService.currentVolume.clamp(0.0, maxVolume);
+    final initialVolume = playerService.currentVolume;
     final isAvailable = playerService.isVolumeAvailable;
     double currentVolume = initialVolume;
     return StatefulBuilder(
@@ -78,8 +77,8 @@ class VolumeBottomSheet extends StatelessWidget {
                     child: Slider(
                       value: currentVolume,
                       min: 0.0,
-                      max: maxVolume,
-                      divisions: (maxVolume * 100).round(),
+                      max: 1.0,
+                      divisions: 100,
                       label: '${(currentVolume * 100).round()}%',
                       onChanged: isAvailable
                           ? (value) {
-- 
2.55.0

