From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: fukiame <fukiame@proton.me>
Date: Fri, 31 Jul 2026 21:00:26 +0700
Subject: [PATCH] Revert "Disable release minification and shrink resources"

This reverts commit 323b3dee8f4ad95314bccf93a619688c769bf8d5.

diff --git a/android/app/build.gradle.kts b/android/app/build.gradle.kts
index d4c256170..71c78351d 100644
--- a/android/app/build.gradle.kts
+++ b/android/app/build.gradle.kts
@@ -78,8 +78,6 @@ android {
     buildTypes {
         release {
             signingConfig = signingConfigs.getByName("release")
-            isMinifyEnabled = false
-            isShrinkResources = false
         }
     }
 }
diff --git a/android/gradle.properties b/android/gradle.properties
index 3a13d188d..db7a19eff 100644
--- a/android/gradle.properties
+++ b/android/gradle.properties
@@ -6,7 +6,3 @@ android.defaults.buildfeatures.buildconfig=true
 android.builtInKotlin=false
 # This newDsl flag was added automatically by Flutter migrator
 android.newDsl=false
-# This builtInKotlin flag was added automatically by Flutter migrator
-android.builtInKotlin=false
-# This newDsl flag was added automatically by Flutter migrator
-android.newDsl=false
-- 
2.55.0

