From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: fukiame <fukiame@proton.me>
Date: Sat, 1 Aug 2026 08:35:36 +0700
Subject: [PATCH] fixup! lib: settings: divide usb and bt

Signed-off-by: fukiame <fukiame@proton.me>

diff --git a/lib/features/settings/screens/settings_screen.dart b/lib/features/settings/screens/settings_screen.dart
index d31657f7..ec09f0ff 100644
--- a/lib/features/settings/screens/settings_screen.dart
+++ b/lib/features/settings/screens/settings_screen.dart
@@ -77,6 +77,7 @@ class _SettingsScreenState extends State<SettingsScreen> {
                           onTap: () =>
                               _navigate(context, const Uac2SettingsScreen()),
                         ),
+                        const SettingsDivider(),
                         _CategoryTile(
                           title: 'Bluetooth',
                           subtitle: 'Configure Bluetooth devices',
-- 
2.55.0

