From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: fukiame <fukiame@proton.me>
Date: Sat, 1 Aug 2026 14:09:56 +0700
Subject: [PATCH] rust: release profile configs

Signed-off-by: fukiame <fukiame@proton.me>

diff --git a/rust/Cargo.toml b/rust/Cargo.toml
index 69f8cae6..9243bda8 100644
--- a/rust/Cargo.toml
+++ b/rust/Cargo.toml
@@ -3,6 +3,11 @@ name = "rust_lib_flick_player"
 version = "0.1.0"
 edition = "2021"
 
+[profile.release]
+strip = "symbols"
+opt-level = "s"
+lto = "fat"
+
 [lib]
 crate-type = ["cdylib", "staticlib"]
 
-- 
2.55.0

