From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: fukiame <fukiame@proton.me>
Date: Sat, 1 Aug 2026 13:37:47 +0700
Subject: [PATCH] isf

Signed-off-by: fukiame <fukiame@proton.me>

diff --git a/lib/features/player/widgets/animated_song_scene.dart b/lib/features/player/widgets/animated_song_scene.dart
index c3b7821c..11ee434f 100644
--- a/lib/features/player/widgets/animated_song_scene.dart
+++ b/lib/features/player/widgets/animated_song_scene.dart
@@ -109,9 +109,7 @@ class AnimatedSongScene extends StatelessWidget {
               );
             },
             transitionBuilder: (Widget child, Animation<double> animation) {
-              final slideOffset = isFullView
-                  ? const Offset(0, 0.12)
-                  : const Offset(0, 0.03);
+              final slideOffset = const Offset(0, 0.03);
               return SlideTransition(
                 position: Tween<Offset>(begin: slideOffset, end: Offset.zero)
                     .animate(
-- 
2.55.0

