From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: fukiame <fukiame@proton.me>
Date: Sat, 1 Aug 2026 19:14:17 +0700
Subject: [PATCH] phat

Signed-off-by: fukiame <fukiame@proton.me>

diff --git a/rust/Cargo.toml b/rust/Cargo.toml
index bdad920d..9243bda8 100644
--- a/rust/Cargo.toml
+++ b/rust/Cargo.toml
@@ -6,7 +6,7 @@ edition = "2021"
 [profile.release]
 strip = "symbols"
 opt-level = "s"
-lto = "thin"
+lto = "fat"
 
 [lib]
 crate-type = ["cdylib", "staticlib"]
-- 
2.55.0

