From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: fukiame <fukiame@proton.me>
Date: Sat, 1 Aug 2026 20:22:43 +0700
Subject: [PATCH] no

Signed-off-by: fukiame <fukiame@proton.me>

diff --git a/rust/Cargo.toml b/rust/Cargo.toml
index 9243bda8..5be794f7 100644
--- a/rust/Cargo.toml
+++ b/rust/Cargo.toml
@@ -5,7 +5,7 @@ edition = "2021"
 
 [profile.release]
 strip = "symbols"
-opt-level = "s"
+opt-level = 3
 lto = "fat"
 
 [lib]
-- 
2.55.0

