From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: fukiame <fukiame@proton.me>
Date: Tue, 28 Jul 2026 01:57:48 +0700
Subject: [PATCH] quet

Signed-off-by: fukiame <fukiame@proton.me>

diff --git a/lib/features/player/widgets/animated_song_scene.dart b/lib/features/player/widgets/animated_song_scene.dart
index 7033f4469..6816bd633 100644
--- a/lib/features/player/widgets/animated_song_scene.dart
+++ b/lib/features/player/widgets/animated_song_scene.dart
@@ -550,9 +550,10 @@ class AnimatedSongScene extends StatelessWidget {
             size: context.responsive(12.0, 13.0, 14.0),
             color: Colors.white.withValues(alpha: highlighted ? 0.96 : 0.7),
           ),
+	  if (count > 0) ...[
           SizedBox(width: context.responsive(4.0, 5.0, 6.0)),
           Text(
-            count > 0 ? 'queue $count' : 'queue',
+            '$count',
             style: TextStyle(
               fontFamily: 'ProductSans',
               fontSize: context.responsive(10.0, 11.0, 12.0),
@@ -560,6 +561,7 @@ class AnimatedSongScene extends StatelessWidget {
               color: Colors.white.withValues(alpha: highlighted ? 0.96 : 0.8),
             ),
           ),
+	  ],
         ],
       ),
     );
-- 
2.55.0

