From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: fukiame <fukiame@proton.me>
Date: Mon, 10 Aug 2026 12:22:11 +0700
Subject: [PATCH] li

Signed-off-by: fukiame <fukiame@proton.me>

diff --git a/lib/features/player/widgets/animated_song_scene.dart b/lib/features/player/widgets/animated_song_scene.dart
index b84bd717..535296de 100644
--- a/lib/features/player/widgets/animated_song_scene.dart
+++ b/lib/features/player/widgets/animated_song_scene.dart
@@ -527,7 +527,7 @@ class AnimatedSongScene extends StatelessWidget {
     return Column(
       crossAxisAlignment: CrossAxisAlignment.start,
       children: [
-        Row( children: Expanded( child: const SizedBox.shrink(), ), ),
+        Row( children: [ Expanded( child: const SizedBox.shrink(), ), ], ),
         if (artworkCardShowTitle)
           Text(
             song.title,
-- 
2.55.0

