From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: fukiame <fukiame@proton.me>
Date: Mon, 10 Aug 2026 07:38:34 +0700
Subject: [PATCH] 50%

Signed-off-by: fukiame <fukiame@proton.me>

diff --git a/lib/features/player/widgets/animated_song_scene.dart b/lib/features/player/widgets/animated_song_scene.dart
index 350e9d66..5bc3cc4d 100644
--- a/lib/features/player/widgets/animated_song_scene.dart
+++ b/lib/features/player/widgets/animated_song_scene.dart
@@ -654,7 +654,6 @@ class AnimatedSongScene extends StatelessWidget {
             (artworkCardShowTitle || artworkCardShowArtist) &&
             (hasAlbum)) ...[
           SizedBox(height: artistToAlbumSpacing),
-          if (hasAlbum)
             GestureDetector(
               onTap: () => onNavigateToAlbumDetail(song),
               child: Container(
@@ -663,10 +662,10 @@ class AnimatedSongScene extends StatelessWidget {
                   vertical: albumVerticalPadding,
                 ),
                 decoration: BoxDecoration(
-                  color: Colors.white.withValues(alpha: 0.06),
+                  color: Colors.white.withValues(alpha: 0.03),
                   borderRadius: BorderRadius.circular(999),
                   border: Border.all(
-                    color: Colors.white.withValues(alpha: 0.1),
+                    color: Colors.white.withValues(alpha: 0.05),
                   ),
                 ),
                 child: Text(
-- 
2.55.0

