From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: fukiame <fukiame@proton.me>
Date: Tue, 11 Aug 2026 14:59:45 +0700
Subject: [PATCH] lib: albums: detail: drop songs count

track count alr there

Signed-off-by: fukiame <fukiame@proton.me>

diff --git a/lib/features/albums/screens/album_detail_screen.dart b/lib/features/albums/screens/album_detail_screen.dart
index 35ddaf26b..73531a226 100644
--- a/lib/features/albums/screens/album_detail_screen.dart
+++ b/lib/features/albums/screens/album_detail_screen.dart
@@ -510,7 +510,7 @@ class _AlbumDetailScreenState extends ConsumerState<AlbumDetailScreen>
               ),
               const SizedBox(height: 4),
               Text(
-                '${widget.albumArtist} • ${widget.songs.length} songs',
+                '${widget.albumArtist}',
                 style: Theme.of(context).textTheme.bodyMedium?.copyWith(
                   color: context.adaptiveTextSecondary,
                 ),
-- 
2.55.0

